cluster/gce/config-test.sh

```diff
@@ -156,3 +156,10 @@ HAIRPIN_MODE="${HAIRPIN_MODE:-promiscuous-bridge}" # promiscuous-bridge, hairpin
 
 # Optional: if set to true, kube-up will configure the cluster to run e2e tests.
 E2E_STORAGE_TEST_ENVIRONMENT=${KUBE_E2E_STORAGE_TEST_ENVIRONMENT:-false}
+
+# A list of images to pre-pull for the e2e, #25277
+function get_image_list {
+  grep -Iiroh "gcr.io/google_.*" "${KUBE_ROOT}/test/e2e" | sed -e "s/[,\")}]//g" | awk '{print $1}' | sort | uniq | tr '\n' ' '
```

Comment:
> I'm intentionally not engineering against injection attacks and the like, because I don't think it matters much for this case.

Since this comment gets into the nits of how we're parsing images, I'm going to clarify that line.

Obviously someone can check in code that does: gcr.io/google_containers;imagename;docker run -it busybox /bin/bash sleep 10000. I don't think that's worth engineering against. 

> If the test is written to pull an image whose name and tag are constructed with arguments, this would be broken, right?

There are none, I fixed the one i found. 

>Can we explicitly whitelist the test image somewhere, and use that whitelist to prepull?

I don't want to do this. It's a maintainence headache, seriously. 

>The generated list doesn't include critical daemonset images, such as kube-proxy, fluentd, etc. We do have flakiness introduced by those images too.

I'm specifically not targetting those custom images. 100% of flake we see today is not from those.